pkg/version.mk

```diff
@@ -15,6 +15,10 @@ define VERSION_LDFLAGS
 $(shell $(CMD_LINKFLAGS) -pkg=$(if $(1),$(1),$(PWD)))
 endef
 
+define VERSION_CUSTOM_FLAGS
```

Comment:
don't think we need this, that was handled with the change to VERSION_LDFLAGS_VENDOR below

Reply:
Ah, this came from my last changes, un-done